CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to insta and cargo-insta are documented here.
 
 ## 1.40.0
 
+- Inline snapshots now use with the required number of `#` to escape the
```

Comment:
The beginning of this sentence has a weird grammar.

Reply:
Thanks, fixed